CONTRIBUTING.md

```diff
@@ -143,18 +142,34 @@ create_sfp(title = "titel van het protocol",
            render = FALSE)
 ```
 
+Alternatively, for a project-specific protocol:
 
-## Translations of protocols
+```r
+library(protocolhelper)
+create_spp(title = "Bodemstalen nemen", 
+           subtitle = "", 
+           short_title = "bodemstalen", 
+           authors = c("Jon Beton", "Jef Plastiek"), 
+           date = Sys.Date(), 
+           reviewers = c("reviewer1, reviewer2"), 
+           file_manager = "manager",
+           project_name = "mne",
+           language = "nl",
+           version_number = "2021.01",
+           render = FALSE)
+```
+
+For project-specific protocols, `protocolhelper::add_subprotocol()` can be used subsequently to add (sections of) published standard field protocols as subprotocols.
```

Comment:
Best add that a specific version needs to be set for each subprotocol (it should not evolve dynamically within a project protocol version). Optionally, project-specific parameters can be passed.